README.md

```diff
@@ -217,5 +217,21 @@ In the save step of example 2 above we included exporting the compiler flag `LIB
 
 MaxText supports training and inference of various open models. Follow user guides under [getting started](https://github.com/google/maxtext/tree/main/getting_started) section to know more. 
 
-* [Gemma](https://ai.google.dev/gemma): a family of open-weights Large Language Model (LLM) by [Google DeepMind](https://deepmind.google/), based on Gemini research and technology. \
-You can run decode and finetuning using instructions mentioned [here](https://github.com/google/maxtext/blob/main/getting_started/Run_Gemma.md).
+### Gemma
+[Gemma](https://ai.google.dev/gemma) is a family of lightweight, state-of-the art open models built from research and technology that we used to create the Gemini models. To get started on decoding and finetuning of Gemma, you will first need to download weights from [kaggle](https://www.kaggle.com/models/google/gemma/frameworks/maxText).
+
+Following the instructions at [kaggle](https://www.kaggle.com/models/google/gemma/frameworks/maxText) will let you download Gemma model weights along with its tokenizer, convert the orbax checkpoints to be compatible with MaxText and upload them to a GCS bucket. You will have to consent to license for Gemma to download weights using your kaggle account's [API credentials](https://github.com/Kaggle/kaggle-api?tab=readme-ov-file#api-credentials).
+
+You can run decode and finetuning using instructions mentioned in the test script [end_to_end/test_gemma.sh](https://github.com/google/maxtext/blob/main/end_to_end/test_gemma.sh).
+
+
+#### Performance
+
+Model Flop utilization for training on v5e and v5p TPUs.
+
+
+| Model    | v5e-256 (bf16) | v5p-128 (bf16) | v5e-256 (int8) | v5p-128 (int8) |
+| -------- | -------------- | -------------- | -------------- | -------------- |
+| Gemma-2b | 58.21%         | 55.36%         | 64.68%         | 67.80%         |
```

Comment:
How many digits do you believe these results would repro to? I believe not the 100th and not hte decimal. Let's remove as integer percentages unless you feel otherwise.